daemon/daemon_linux.go

```diff
@@ -54,7 +54,6 @@ func (daemon *Daemon) cleanupMountsFromReaderByID(reader io.Reader, id string, u
 		return fmt.Errorf("Error cleaning up mounts:\n%v", strings.Join(errors, "\n"))
 	}
 
-	logrus.Debugf("Cleaning up old mountid %v: done.", id)
```

Comment:
Why was it removed? It's practical to have the matching done from the start printed in `cleanupMountsByID` when trying to figure out timing in daemon log :)